docs/data-guidelines.md

```diff
@@ -256,3 +257,26 @@ If the table indicates an initial version of "1" and an information source says
 If you're adding new data for Node.js, use `0.10.0` or later. If a feature was added in a version before `0.10.0`, use `0.10.0` anyway.
 
 This guideline was proposed in [#6861](https://github.com/mdn/browser-compat-data/pull/6861).
+
+## Mixins
+
+Mixins appear in WebIDL and in specifications to define Web APIs. For web developers, they aren't observable directly; they act as helpers to avoid repeating API definitions. Don't add mixins to BCD where they do not already exist.
+
+For example, [`HTMLHyperlinkElementUtils`](https://html.spec.whatwg.org/multipage/links.html#htmlhyperlinkelementutils) is a mixin defined in the HTML specification.
+
+Members of this mixin are available to `HTMLAnchorElement` and `HTMLAreaElement`, that's where BCD exposes them. Add mixin members to BCD in one of these ways:
+
+1. For smaller mixins, add members of `HTMLHyperlinkElementUtils` directly to the `api/HTMLAnchorElement.json` and `api/HTMLAreaElement.json` files as if they were regular members of these interfaces.
+
+2. For larger mixins, create a file in the `api/_mixins/` folder and indicate for which interface they are using file names like: `HTMLHyperlinkElementUtils__HTMLAnchorElement.json` and `HTMLHyperlinkElementUtils__HTMLAreaElement.json`.  
```

Comment:
Any particular reason there should be two underscores in the filenames? Is it because some Web IDL interface names have underscores in them? I'd just ignore that, or maybe use a separator that can't appear in interface/mixin names, like a dash.

Reply:
Originally, I proposed `HTMLHyperlinkElementUtils-for-HTMLAnchorElement.json`, but `HTMLHyperlinkElementUtils-HTMLAnchorElement.json` would be fine with me, too. @ddbeck any reason you proposed two underscores?

Reply:
I just thought it made the IDL part stand out from the the non-IDL part better, as compared to one. I was just throwing an idea out there. I don't feel strongly about how it's done.

Reply:
I guess two underscores is fine then. It does separate them nicely.